src/osd/osd_types.cc

```diff
@@ -3761,7 +3761,7 @@ void pg_create_t::generate_test_instances(list<pg_create_t*>& o)
 
 void pg_hit_set_info_t::encode(bufferlist& bl) const
 {
-  ENCODE_START(1, 1, bl);
+  ENCODE_START(2, 2, bl);
   ::encode(begin, bl);
   ::encode(end, bl);
   ::encode(version, bl);
```

Comment:
that's what i expect: to upgrade existing OSDs from `using_localtime` to `using_gmt` automatically.